src/install/lockfile.zig

```diff
@@ -112,7 +112,11 @@ const Stream = std.io.FixedBufferStream([]u8);
 pub const default_filename = "bun.lockb";
 
 pub const Scripts = struct {
-    const StringArrayList = std.ArrayListUnmanaged(string);
+    const Entry = struct {
```

Comment:
we almost could add a --filter with this! Just missing another map to say "which package.json name does it belong to"